docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -15,48 +15,241 @@ related:
-    1. In **STORE RELATION**, check the stores you wish the discount to be active in.
-    2. From the **DISCOUNT TYPE** drop-down list, select a cart rule.
-    3. In the **NAME** field, specify the name for the discount.
-    4. _Optional_: in the **DESCRIPTION** field, enter the description of the discount. The description is displayed in the cart, in the **Promotional products** section.
-    5. _Optional_: in the **PRIORITY** field, enter an integer value from `1` to `9999` for the discount priority. For reference information, in the **Create new discount** reference, see [General information tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#general-information-tab).
-    6. Specify if the discount is exclusive. For reference information, in the **Create new discount** reference, see [General information tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#general-information-tab).
-    7. Specify the validity interval (lifetime) of the discount.
-* Click **Next** or select the **Discount calculation** tab to proceed.
+2. On the **Create new discount** page, in the **General Information** tab, do the following:
+   1. In **STORE RELATION**, select stores you wish the discount to be active in.
+   2. For **DISCOUNT TYPE**, select **Cart rule**.
+   3. In the **NAME** field, specify the name for the discount.
+   4. Optional: in the **DESCRIPTION** field, enter a summary that explains the discount.
+   5. Optional: in the **PRIORITY** field, enter an integer value from `1` to `9999` for the discount priority.
+   6. Specify if the discount is either **NON-EXLUSIVE** or **EXCLUSINVE**.
+   7. In **VALID FROM** and **VALID TO**, Specify the validity interval (lifetime) of the discount.
+   8. To proceed to the **Discount calculation** tab, click **Next**.
+3. On the **Discount calculation** tab, do the following:
+   1. In the **CALCULATOR TYPE** drop-down, select either **Percentage** or **Fixed amount**.
+
+      {% info_block warningBox "Note" %}
 
-* On the **Discount calculation** tab, do the following:
-    1.  In the **CALCULATOR TYPE** drop-down, select either **Percentage** or **Fixed amount**. For reference information, in the **Create new discount** reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
+      The next step varies based on the calculator type that you select:
+      * **Fixed amount**: Enter the amounts (per currency and price mode, if applicable) to be discounted.
+      * **Percentage**: Enter the percent value to be discounted.
 
-        {% info_block warningBox "Note" %}
+      {% endinfo_block %}
 
-        The next step varies based on the selected calculator type:
+   2. Select the **DISCOUNT APPLICATION TYPE** and define which products are eligible for the discount.
```

Comment:
Separate the second part of this sentence and try to use the approach from the previous comment.